.github/CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the
+# repo. Unless a later match takes precedence, these owners will be
+# requested for review when someone opens a pull request.
+* @dav3r @felddy @jsf9k @mcdonnnj
```

Comment:
I don't believe this code currently has any owners in the cisagov org.  Definitely agree that our team should own the `.github/` directory.
```suggestion
```